pokedex2.0/src/About.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
You also need to import { Component } if you aren't going to use it as React.Component like you did in CharacterItem